File: Mock1-DJ-2019/editor.java
import java.io.*;
import java.util.*;

class editor {
    public static void main(String args[]) {
        Scanner sc = new Scanner(System.in);

        long operations = sc.nextLong();

        long num1 = (long) Math.ceil((double)(operations - 2) / 2);
        long num2 = Math.abs((operations - 2) - num1);

        long val;

        if(num1 > num2) {
            val = num1 + num2 * num1;
        } else {
            val = num2 + num1 * num2;
        }
            
        System.out.println(val);
    }
}

File: Mock1-DJ-2019/editor.py
import math

operations = int(input())

num1 = math.ceil((operations - 2) / 2)
num2 = abs((operations - 2) - num1)

if num1 > num2:
    print(num1 + num2 * num1)
else:
    print(num2 + num1 * num2)
